drivers/display/display_ili9340.c

```diff
@@ -8,37 +8,31 @@
 #include "display_ili9340.h"
 #include <drivers/display.h>
 
-#define LOG_LEVEL CONFIG_DISPLAY_LOG_LEVEL
 #include <logging/log.h>
-LOG_MODULE_REGISTER(display_ili9340);
+LOG_MODULE_REGISTER(display_ili9340, CONFIG_DISPLAY_LOG_LEVEL);
 
 #include <drivers/gpio.h>
 #include <sys/byteorder.h>
 #include <drivers/spi.h>
 #include <string.h>
 
+#define RES_X_MAX 320U
+#define RES_Y_MAX 240U
+
 struct ili9340_data {
 #ifdef DT_INST_0_ILITEK_ILI9340_RESET_GPIOS_CONTROLLER
-	struct device *reset_gpio;
+	struct device *rst;
 #endif
-	struct device *command_data_gpio;
```

Comment:
I agree that it was to long and you probably changed it to resolve coding style issues. But it would be better to keep reset_gpio + cs_gpio, and change command_data_gpio to cmd_gpio.

You put to many different changes in one commit. Sorry, but it is hard to review.

Reply:
Form my point of view it is excessive, vars are used with gpio_* functions indicating what kind of device is it. In general in makes sense, but here we have a particular driver where "cmd" is name of pin (cnd - command not data whoul be even better) .

So sorry, there are a lot of changes, which depend on each other